tests/test_loadref_hsc.py

```diff
@@ -141,8 +156,15 @@ def test_fgcmLoadReferenceOtherFilters(self):
         config.colorterms.data['ps1*'].data['HSC-I2'].c1 = -0.200406
         config.colorterms.data['ps1*'].data['HSC-I2'].c2 = -0.013666
 
-        butler = dafPersist.Butler(self.inputDir)
-        loadCat = fgcmcal.FgcmLoadReferenceCatalogTask(butler=butler, config=config)
+        butler = lsst.daf.butler.Butler(self.repo, instrument='HSC', collections=['HSC/testdata',
+                                                                                  'refcats/gen2'])
```

Comment:
`refcats/gen2`?!

Reply:
That's what they get called because the test repo is converted from gen2 to gen3.  🤷